docs/repositories/issues.md

```diff
@@ -53,7 +53,9 @@ You can define one or more of the following filters:
 
 ## Ignoring and managing issues
```

Comment:
Fair enough, but it's an important option that the menu offers. I reordered the options to match the new order on the menu, but it doesn't make much sense to title it "Copying links to an managing issues" 😆 

Could we invert here and start with the more generic "managing"? I'd still keep the old ID in case we have inbound links to this section.

```suggestion
## Managing and ignoring issues {: id="ignoring-and-managing-issues"}
```

Reply:
This is something that I noticed - because it was not a perfect match with the order on UI - but, honestly, I believe the current title, **Ignoring and managing issues**, still works better from a writing perspective. Because we first have a relevant and specific action - ignoring issues - and the "other actions" come after (managing stuff), which could be better for the findability of relevant operations.

Let's keep it as is. 👍
Maybe it's the product that needs some improvement, ordering the options by relevance... 😉